openllm-python/src/openllm/conversion/__init__.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import annotations
+import typing as t
+
+def openllm_to_openai_config(input_dict: dict[str, t.Any], default_config: dict[str, t.Any]) -> dict[str, t.Any]:
```

Comment:
Let's just move this file into `protocol/utils.py`, or you should make this a function of `LLMConfig`. This can be found under `openllm_core`.

```python

openai_config = openllm.AutoConfig.for_model('llama').to_openai_config()